sigstore/_internal/fulcio/client.py

```diff
@@ -262,15 +273,16 @@ def _opaque16(value: bytes) -> bytes:
             raw_sct = _opaque16(_opaque16(raw_sct_list_bytes))
 
             sct = precert_scts_extension[0]
-        except ExtensionNotFound:
+        else:
             # If we don't have any embedded SCTs, then we might be dealing
             # with a Fulcio instance that provides detached SCTs.
 
             # The SCT header is a base64-encoded payload, which in turn
             # is a JSON representation of the SignedCertificateTimestamp
             # in RFC 6962 (subsec. 3.2).
-            sct_b64 = resp.headers.get("SCT")
-            if sct_b64 is None:
+            try:
+                sct_b64 = resp.json()["signedCertificateDetachedSct"]["signedCertificateTimestamp"]
+            except KeyError:
                 raise FulcioClientError("Fulcio response did not include a SCT header")
```

Comment:
Sorry, one last nit: it looks like the detached SCT is no longer in the header, right? So this exception should probably be tweaked to something like...

```suggestion
                sct_b64 = resp.json()["signedCertificateDetachedSct"]["signedCertificateTimestamp"]
            except KeyError:
                raise FulcioClientError("Fulcio response did not include a detached SCT")
```

Reply:
Yep, good catch! Fixed